moderation/Slowmode.go.tmpl

```diff
@@ -0,0 +1,60 @@
+{{/*
+	This command helps using a slowmode. it deletes a user's message if their slowmode isnt over people who you want can bypass slowmode as well as set it 
+ 
+	Usage: "-slowmode on (time in seconds) / -slowmode off "
+ 
+	Recommended trigger: ".*"
+	Trigger type: Regex
+ 
+	Credits:
+	Sponge :D
+*/}}
+ 
+{{/* CONFIGURATION VALUES START */}}
+{{$bypassperms := "ManageMessages"}}{{/* perms needed to bypass slowmode */}}
+{{$usageperms := "ManageMessages"}}{{/* perms needed for slowmode commands */}}
+{{/* put in the permissions you want the use to have in order to use or bypass the command */}}
+{{/* available perms: Administrator, ManageServer, ReadMessages, SendMessages, SendTTSMessages, ManageMessages, EmbedLinks, AttachFiles, ReadMessageHistory, MentionEveryone, VoiceConnect, VoiceSpeak, VoiceMuteMembers, VoiceDeafenMembers, VoiceMoveMembers, VoiceUseVAD, ManageNicknames, ManageRoles, ManageWebhooks, ManageEmojis, CreateInstantInvite, KickMembers, BanMembers, ManageChannels, AddReactions, ViewAuditLogs  */}}
+{{/* CONFIGURATION VALUES END */}}
+ 
+{{/* ACTUAL CODE */}}
+{{ $usageaccess := false }}
+{{ $bypassaccess := false }}
+{{ $viewperms := (split (index (split (exec "viewperms") "\n") 2) ", ") }}
+{{ if (in $viewperms $usageperms)}}{{ $usageaccess = true }}{{end}}
+{{ if (in $viewperms $bypassperms)}}{{ $bypassaccess = true }} {{ end }}
+{{if $usageaccess}}
+{{ if $matches := reFindAllSubmatches `\A-slowmode on (\d+)` .Message.Content }}
+{{$slowmodeduration := (index $matches 0 1)}}
+{{$db := (dbGet 660 "channels").Value}}
+{{with $db}}
+{{$value := sdict .}}
+{{$value.Set (str $.Channel.ID) (str $slowmodeduration)}}
```

Comment:
Don't stringify, use the int/float directly. Goes for both ID + slowmode duration.